app/package.json

```diff
@@ -9,6 +9,37 @@
     "type": "git",
     "url": "https://github.com/jupyterlite/jupyterlite"
   },
+  "scripts": {
+    "clean": "rimraf build lab/build retro/build",
+    "build": "webpack",
+    "build:prod": "yarn clean && yarn build --mode=production",
+    "watch": "webpack --config webpack.config.watch.js"
+  },
+  "jupyterlite": {
+    "liteApps": [
```

Comment:
No strong opinion, but maybe just `apps` would be enough since this is already inside `jupyterlite`?

Reply:
[695ccda](https://github.com/jupyterlite/jupyterlite/pull/474/commits/695ccda5b5f73757619b35503f7f14b77af9a704)